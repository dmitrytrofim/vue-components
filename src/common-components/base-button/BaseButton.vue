<template>
  <button class="test12"><slot></slot></button>
</template>

<script lang="ts" setup></script>

<style scoped></style>
