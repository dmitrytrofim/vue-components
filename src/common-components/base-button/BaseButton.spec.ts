import { mount } from '@vue/test-utils';
import BaseButton from '@/common-components/base-button/BaseButton.vue';

describe('BaseButton snapshot', () => {
  test('matches snapshot', () => {
    const wrapper = mount(BaseButton, {
      slots: {
        default: 'Click me!',
      },
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
