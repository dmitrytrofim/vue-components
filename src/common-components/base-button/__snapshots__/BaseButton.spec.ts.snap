// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseButton snapshot matches snapshot 1`] = `"<button class="test12">Click me!</button>"`;
